app/dashboards/event_dashboard.rb

```diff
@@ -69,6 +71,7 @@ class EventDashboard < Administrate::BaseDashboard
     :name,
     :description,
     :attendee_information,
+    :remote_url,
```

Comment:
we already have the <attendee_information> attribute that is displayed as markdown, thus it can contain urls. do you think it's beneficial to have another attribute specific to this usecase?

Reply:
I thought about that, too. The benefit of having a specific URL attribute is that you can show a "Connect" link/button in the event page and that makes it more usable IMHO (it does so only for users that are attending the meeting)

Reply:
ok good, then let's go ahead with this.

Reply:
Thanks!!!